index.html

```diff
@@ -4,42 +4,40 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/styles.css" />
     <title>Document</title>
 </head>
 <body>
     <header class="header">
```

Comment:
you don't have to give your header the class header
```suggestion
    <header>
```